step_two_utils.py

```diff
@@ -0,0 +1,222 @@
+import json
+import os
+import requests
+
+import utils
+
+log_dir = os.environ.get('PY_GOOGLE_AUTH_LOG_PATH')
+
+
+def two_step_login_with_prompt(session, payload, q_params, url_to_challenge_signin):
+    '''
+    Method for two step authentication with Google prompt.
+    Collects a key and txId from the `form_html` to create a payload to send to next page,
+    which awaits use confirmation from prompt.
+    When User confirms, it makes a POST call to final challenge url to verify the request.
+    If request was accepted, it returns the google play home page with user logged in.
+
+    `url_to_challenge_signin`: the url to make final call to verify request status
+    and allow/deny login.
+    `await_url`: url to make a POST call to check if a user responded on prompt.
+    `q_params`: These are two query parameters used to call `await_url`.
+    `payload`: payload to send with POST request, i.e. cookies, tokens etc. more details in
+    `utils.make_payload` function.
+    '''
+    error = None
+
+    await_url = "https://content.googleapis.com/cryptauth/v1/authzen/awaittx?alt=json&key="
+
+    # headers are necessary to specify the referer and content type else request fails.
+    headers = {"Referer": url_to_challenge_signin, "Content-Type": "application/json"}
+    key = q_params['key']
+    txId = q_params['txId']
```

Comment:
What is txId it is not clear to me, if you are opensourcing it please take care of this type of things. I know little bit about this project and this is not clear to me how will this is understandable for others?